modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/wstep/impl/CertificateEnrollmentServiceImpl.java

```diff
@@ -246,7 +246,7 @@ public String prepareWapProvisioningXML(
         }
         DocumentBuilderFactory domFactory = DocumentBuilderFactory.newInstance();
         DocumentBuilder builder;
-        String wapProvisioningString = null;
+        String wapProvisioningString;
```

Comment:
What's meant by "wapProvisioningString"? Consider using some other appropriate variable name depending on the context in which the same is used.